service/app/controllers/feed/validator/FeedRequestValidator.java

```diff
@@ -0,0 +1,13 @@
+package controllers.feed.validator;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.common.exception.ProjectCommonException;
+
+/** This call will validate the Feed API request */
+public class FeedRequestValidator {
+  public static void userIdValidation(String accessTokenUserId, String requestUserId) {
+    if (!StringUtils.endsWithIgnoreCase(accessTokenUserId, requestUserId)) {
```

Comment:
why u want to match prefix with  equal ignore case?
if both are null then it will be true that condition need to be handle